Procfile

```diff
@@ -1,2 +1,3 @@
-web: bundle exec puma -C config/puma.rb
+web: BIND=0.0.0.0 bundle exec puma -C config/puma.rb
+#web: BIND=0.0.0.0 node ./streaming # for streaming app
```

Comment:
Why commented out, and why not call it "streaming"?

Reply:
Heroku only routes requests from the Internet at ports 80 or 443 to "web" dynos.

To run the streaming server on Heroku, a separate app that shares Postgres and Redis with the main app is needed. On that app, the node process needs to run on "web" dynos.

Reply:
Okay. I would format it like this:

```Procfile
web: BIND=0.0.0.0 bundle exec puma -C config/puma.rb
worker: bundle exec sidekiq

# For the streaming API, you need a separate app that shares Postgres and Redis with
# the main one, and runs this dyno:
# web: BIND=0.0.0.0 node ./streaming 
```

Reply:
I pushed d7d15d4 so that the admin doesn't have to edit the `Procfile`. How does this look to you?